python/mpt/README.md

```diff
@@ -0,0 +1,173 @@
+# Ethereum MPT Library
+
+A Python library for working with Ethereum's Merkle Patricia Tries (MPT).
+
+## Overview
+
+This library provides utilities to decode and work with Ethereum's trie
+structures, used in the state, storage, and transaction tries.
+
+## Features
+
+- Decode RLP-encoded MPT nodes
+- Support for all node types (Branch, Extension, Leaf)
+- Load Ethereum trie structures from JSON files
+- Helper functions for nibble path manipulation
+
+## Quick Start
+
+````python
+from mpt.ethereum_tries import EthereumTries
+from pathlib import Path
```

Comment:
lint

Reply:
how do you apply lint on readme python block?

Reply:
i applied this order
1/ core lib imports

2/ external imports

3/ internal imports

Reply:
I don't have automatic way to do it, you can write it first in python elsewhere?